### PR TITLE
MF-141: Update Programs Navigation and Structure

### PR DESCRIPTION
**Pull request by denniskigen**
This PR modifies the programs widget so that it corresponds with the routing and navigation structure specified in https://issues.openmrs.org/projects/MF/issues/MF-141.

Screenshot 2020-03-20 at 13 07 05

Screenshot 2020-03-20 at 13 07 14

### CODE REVIEW

**Comment by jonathandick** [comment]
This is good work @denniskigen.

For now, please don't update configs for views. The conditions tab should be remain the same.

File: src/widgets/programs/programs-overview.component.tsx [hunk 1]
```diff
@@ -35,7 +39,7 @@ export default function Programs(props: ProgramsOverviewProps) {
   return (
     <SummaryCard
       name={t("care programs", "Care Programs")}
-      link={`/patient/${patientUuid}/chart/programs`}
+      link={`${props.basePath}`}
```
Comment (jonathandick) [inline src/widgets/programs/programs-overview.component.tsx]: This should be programsPath not props.basePath

**Review by jonathandick — CHANGES_REQUESTED** [review]
One small change and then we can merge.

File: src/widgets/programs/programs-overview.component.tsx (hunk 1 above)
Comment (denniskigen) [reply src/widgets/programs/programs-overview.component.tsx]: Fixed

**Comment by denniskigen** [comment]
> This is good work @denniskigen.
> 
> For now, please don't update configs for views. The conditions tab should be remain the same.

Ok, this was only for the purposes of getting these screenshots

**Pull request merged**